core/modules/wmgr/python/dbMgr.py

```diff
@@ -916,7 +902,7 @@ def getIndex(dbName, tblName, chunkId=None):
         allRows = dbConn.execute(query)
 
         if allRows.keys():
-            descr = [dict(name=d[0], type=_typeCode2Name(d[1])) for d in allRows.keys()]
+            descr = [dict(name=d[0], type=utils.typeCode2Name(dbConn, d[1])) for d in allRows.keys()]
```

Comment:
Clearly integration tests are not exercising this code... can you think of a way to address that?